src/Akeneo/Pim/Structure/Family/back/ServiceAPI/Query/FamilyQuery.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Akeneo\Pim\Structure\Family\ServiceAPI\Query;
+
+use Webmozart\Assert\Assert;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ *
+ * @immutable
+ */
+final class FamilyQuery
+{
+    /**
+     * @param array|null $includeCodes families will be searched only inside families corresponding to $includeCodes
+     * @param array|null $excludeCodes families corresponding to $excludeCodes will be put out of the search
```

Comment:
```suggestion
     * @param string[]|null $includeCodes families will be searched only inside families corresponding to $includeCodes
     * @param string[]|null $excludeCodes families corresponding to $excludeCodes will be put out of the search
```
Or `array<string>`, not sure which is the prefered/chosen way on backend